src/seedu/addressbook/data/person/Phone.java

```diff
@@ -8,9 +8,9 @@
  */
 public class Phone {
 
-    public static final String EXAMPLE = "123456789";
-    public static final String MESSAGE_PHONE_CONSTRAINTS = "Person phone numbers should only contain numbers";
-    public static final String PHONE_VALIDATION_REGEX = "\\d+";
+    public static final String EXAMPLE = "123456789 or 1234 5678";
+    public static final String MESSAGE_PHONE_CONSTRAINTS = "Person phone numbers should only contain numbers and whitespace(s)";
+    public static final String PHONE_VALIDATION_REGEX = "(\\d|\\s)+";
```

Comment:
I apologise for double review. I just taught it would be good to highlight the line involved using the GitHub build in function so you can see directly the line in which I was referring to.

As stated an alternative (just for general knowledge as there is nothing wrong with using your current syntax),
`public static final String PHONE_VALIDATION_REGEX = "[\\d\\s]+";`